docs/articles/documentation/templates/authentication/role-options.md

```diff
@@ -0,0 +1,33 @@
+### options.preserveUrl
+
+Use this option to control which page is opened after you switch to the role.
+
+By default, TestCafe navigates back to the page that was opened previously to switching to the role.
+Set the `preserveUrl` option to `true` to save the URL to which the browser was redirected after logging in.
+TestCafe will navigate to the saved URL each time after you switch to this role.
+
+This option is useful if you store session-related data (like session ID) in the URL.
```

Comment:
This sentence is not really suited for the docs, I think. We can change it to something like "You can use this .. to store .. data" or "This can be used to store .. data"